Dockerfile

```diff
@@ -51,27 +53,53 @@ RUN cd /build/nginx-clojure-${NGINX_CLOJURE_VERSION} && \
     LEIN_ROOT=1 /build/lein uberjar && \
     mkdir -p /usr/lib/nginx/jars && \
     cp target/nginx-clojure-${NGINX_CLOJURE_VERSION}-standalone.jar \
-	/usr/lib/nginx/jars/nginx-clojure.jar && \
+        /usr/lib/nginx/jars/nginx-clojure.jar && \
     rm -rf $HOME/.m2 $HOME/.lein
 
 # Add config
 COPY conf/nginx.conf /etc/nginx/nginx.conf
 COPY conf/clojure.conf /etc/nginx/conf.d/clojure.conf
+
+# -------------------------------------------------------------------
+# Build the runtime container
+# -------------------------------------------------------------------
+FROM openjdk:8-jre-alpine
+
+ARG NGINX_CLOJURE_VERSION
+ENV NGINX_CLOJURE_VERSION ${NGINX_CLOJURE_VERSION}
+
+RUN apk update --no-cache && \
```

Comment:
The `update` and `upgrade` are redundant. The `apk add --no-cache` command should suffice.